styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java

```diff
@@ -187,6 +187,18 @@ void workflowDeleted(Workflow workflow) {
     workflowRemoveListener.accept(workflow);
   }
 
+  /**
+   * Fast forwards the time without any execution in-between.
+   */
+  void timeJumps(int n, TimeUnit unit) {
```

Comment:
Do you think it's somehow possible to have the distinction even clearer between `timePasses` and `timeJumps` without having to read the javadoc? `jump` compared to `pass` is in the right direction but I would definitely need to read the javadoc to know for sure.

Not a big deal at all, just thought it was unclear.